### PR TITLE
Switch from Google Analytics to GoatCounter

### PR DESCRIPTION
**Pull request by mathcolo**
The time has come.

NOTE: ⚠️ This is against `bus-staging`

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -116,8 +117,11 @@ class App extends React.Component {
       progressBarKey: 0,
     };
 
-    ReactGA.initialize("UA-71173708-2");
-    ReactGA.pageview(props.location.pathname);
+    if(window.location.hostname === PRODUCTION) {
```
Comment (austinjpaul) [inline src/App.js:120]:
```suggestion
    if (window.location.hostname === PRODUCTION) {
```

File: src/App.js [hunk 2]
```diff
@@ -328,7 +332,12 @@ class App extends React.Component {
           route: configuration.line,
         });
       }
-      ReactGA.pageview(window.location.pathname + window.location.search);
+
+      if (window.location.hostname === PRODUCTION) {
+        window.goatcounter.count({
+          path: window.location.pathname + window.location.search,
+        });
```
Comment (austinjpaul) [inline src/App.js:339]: trying to wrap my head around this API. It seems like pathname + location is the default? is there a way to let goatcounter get that info on its own so we don't have to specify?

**Review by austinjpaul — APPROVED** [review]
THANK YOU THANK YOU!
This looks awesome. Thrilled to ditch GA.
2 questions:
- Do we need to update package-lock bc package changed?
- have we tested this?

File: src/App.js (hunk 2 above)
Comment (mathcolo) [reply src/App.js:339]: I don't think so; it requires a `path`: https://www.goatcounter.com/help/path

Comment (austinjpaul) [reply src/App.js:339]: Fair enough- this is very clear

**Pull request merged**